mlflow/models/model.py

```diff
@@ -108,7 +108,7 @@ def save(self, path):
     def load(cls, path):
         """Load a model from its YAML representation."""
         if os.path.isdir(path):
-            path = os.path.join(path, MLMODEL_FILE_NAME)
+            path = os.path.normpath(os.path.join(path, MLMODEL_FILE_NAME))
```

Comment:
Must call `os.path.normpath()` everywhere to prevent having a file path with both `/`s and `\`s in it. Great!

Reply:
Seriously? But doesn't CI/CD already work for Python on Windows?

Reply:
I think there must have been some corner case that slipped through the crack in existing py tests on Windows... The places where both `/` and `\` were both present (as I remember) were such that part of the path came from an env variable containing `/` and then `os.path.join()` introduced `\`s (or some other combination of those...). There was also one place where `/` was hard-coded in the some `.py` file which was unfortunate.

I think `os.path.normpath()` is good to have everywhere because it allows us to normalize path separators from env variables or other user inputs, avoiding both `\` and `/` being present in the result.